documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1047 @@
+<6> Specify the API end point URL, for example, `https://<engine_host>/ovirt-engine/api`.
+endif::[]
+ifdef::ova[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: ova
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  url: <nfs_server:/nfs_path> <2>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> where: `nfs_server` is an IP or hostname of the server where the share was created and `nfs_path` is the path on the server where the OVA files are stored.
+endif::[]
+ifdef::ostack[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openstack
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  domainName: <domain_name>
+  projectName: <project_name>
+  regionName: <region name> <5>
+  cacert: | <6>
+    <ca_certificate>
+  url: <api_end_point> <7>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {osp} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
+<5> Specify the name of the {osp} region.
+<6> Enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`. Note that if you set `insecureSkipVerify` to `false` and you do not set  `cacert`, {project-short} attempts to use the system CA.
+<7> Specify the API end point URL, for example, `https://<identity_service>/v3`.
+endif::[]
+
+ifdef::cnv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openshift
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
```

Comment:
here we'll also have `cacert` which is similar to that of RHV and OpenStack

Reply:
Same comment as above -- Here CNV is the source.
Text for callout now reads
 Enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.

Reply:
here it's a CA certificate for the API server of the OpenShift cluster

Reply:
@ahadas  Thursday evening: 

1. would that be "Enter the CA certificate for the API server of the {virt} server unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate."?

2.  Do we need to say anything about the URL for the API server of the OpenShift cluster or is that address known to the user already?